Content.Server/Body/Surgery/SurgerySystem.cs

```diff
@@ -0,0 +1,560 @@
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner),
+                        ("part", target),
+                        ("procedure", operation.Name)),
+                        except: part.Body.Owner);
+                }
+                else
+                {
+                    var id = "surgery-prepare-start-self-no-zone-surgeon-popup";
+                    target.PopupMessage(surgeon.Owner, Loc.GetString(id,
+                        ("item", drapes.Owner),
+                        ("procedure", operation.Name)));
+
+                    id = "surgery-prepare-start-self-no-zone-outsider-popup";
+                    target.PopupMessage(surgeon.Owner, Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner)));
+                }
+            }
+            else
+            {
+                if (IsReceivingSurgeryOnPart(target, out var part, out var body))
+                {
+                    var id = "surgery-prepare-start-surgeon-popup";
+                    body.Owner.PopupMessage(surgeon.Owner, Loc.GetString(id,
+                        ("item", drapes.Owner),
+                        ("target", body.Owner),
+                        ("zone", part.Owner),
+                        ("procedure", operation.Name)));
+
+                    id = "surgery-prepare-start-target-popup";
+                    surgeon.Owner.PopupMessage(body.Owner, Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner),
+                        ("zone", part.Owner)));
+
+                    id = "surgery-prepare-start-outsider-popup";
+                    surgeon.Owner.PopupMessageOtherClients(Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner),
+                        ("target", body.Owner),
+                        ("zone", target)),
+                        except: body.Owner);
+                }
+                else
+                {
+                    var id = "surgery-prepare-start-no-zone-surgeon-popup";
+                    target.PopupMessage(surgeon.Owner, Loc.GetString(id,
+                        ("item", drapes.Owner),
+                        ("target", target),
+                        ("procedure", operation.Name)));
+
+                    id = "surgery-prepare-start-no-zone-target-popup";
+                    surgeon.Owner.PopupMessage(target, Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner)));
+
+                    id = "surgery-prepare-start-no-zone-outsider-popup";
+                    surgeon.Owner.PopupMessageOtherClients(Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner),
+                        ("target", target)),
+                        except: target);
+                }
```

Comment:
I can't read these in-game as they get pushed out at once and look like elven text from lotr.

![image](https://user-images.githubusercontent.com/31366439/134105970-7cfb9c27-37a5-4cfd-87f0-d767a37a6703.png)